signer/client/signer_trust.go

```diff
@@ -180,16 +179,7 @@ func (trust *NotarySigner) ListAllKeys() map[string]string {
 // CheckHealth checks the health of one of the clients, since both clients run
 // from the same GRPC server.
 func (trust *NotarySigner) CheckHealth(timeout time.Duration) error {
-
-	// Do not bother starting checking at all if the connection is broken.
-	if trust.clientConn.State() != grpc.Idle &&
```

Comment:
It looks like `addrConn` is unexported too.  :|  We might not be able to check the connection status here before actually making the request (which will fail if there's no connection, it just might be slower).